### PR TITLE
DAOS-11727 test: pool/management_race.py del_and_recreate_pool intermittent failures

### PR DESCRIPTION
**Pull request by dinghwah**
Description: launch test with 30 repeat loops
Test-repeat: 30
Test-tag: pool_mgmt_race
Required-githooks: true

Signed-off-by: Ding Ho ding-hwa.ho@intel.com

### CODE REVIEW

**Review by daltonbohning — COMMENTED** [review]
If you intend to merge these changes, will you also fix the flake issues in `management_race.py`?

File: src/tests/ftest/pool/management_race.py [hunk 1]
```diff
@@ -65,6 +65,9 @@ def del_recreate_query_and_list_pools(
                             "race condition detected by dmg, retry.. %s", pool_id, error)
                         time.sleep(wait_time)
                         if float(time.time()) - start > max_query_time:
+                            self.info("Log reference (a.b.c.d): \n a: test_thread_number \n"
```
Comment (daltonbohning) [inline src/tests/ftest/pool/management_race.py:68]:
```suggestion
                            self.log.info("Log reference (a.b.c.d): \n a: test_thread_number \n"
```

**Review by daltonbohning — CHANGES_REQUESTED** [review]

File: src/tests/ftest/pool/management_race.py (hunk 1 above)
Comment (dinghwah) [reply src/tests/ftest/pool/management_race.py]: Thanks for the review.
Fixed.

File: src/tests/ftest/pool/management_race.yaml [hunk 2]
```diff
@@ -1,3 +1,8 @@
+"""
```
Comment (shimizukko) [inline src/tests/ftest/pool/management_race.yaml:1]: Do we need copyright for yaml? I don't see it in any other yaml file.

**Review by shimizukko — APPROVED** [review]

File: src/tests/ftest/pool/management_race.yaml (hunk 2 above)
Comment (dinghwah) [reply src/tests/ftest/pool/management_race.yaml:1]: Yes. When I commit this PR, it start showing error about the copyright on the yaml file.

Comment (dinghwah) [reply src/tests/ftest/pool/management_race.yaml]: done.

**Review by shimizukko — APPROVED** [review]

**Pull request merged**